cola/widgets/selectcommits.py

```diff
@@ -56,30 +56,38 @@ def __init__(self, context, model, parent=None, title=None, multiselect=True):
 
         self.commit_text = DiffTextEdit(context, self, whitespace=False)
 
-        self.label = QtWidgets.QLabel()
-        self.label.setText(N_('Revision Expression:'))
+        self.revision_label = QtWidgets.QLabel()
+        self.revision_label.setText(N_('Revision Expression:'))
         self.revision = QtWidgets.QLineEdit()
         self.revision.setReadOnly(True)
```

Comment:
we have a RefLineEdit thing that handles completions for git refs (branch names, tags). We should probably use it for the revision field.

Reply:
Can you explain more about this ?

Reply:
self.revision = QtWidgets.QLineEdit() to self.revision = QtWidgets.RefLineEdit() ?

Reply:
complettion.GitRefLineEdit(context) you mean this ? Let me try it.

Reply:
Added this in my new commit.

Reply:
![image](https://user-images.githubusercontent.com/29145667/111594930-a597f700-87f1-11eb-8ace-9235773f3078.png)